src/components/navigation/Navbar.vue

```diff
@@ -38,11 +38,11 @@
 
 <script lang="ts">
 import Vue from 'vue';
-import { mapState } from 'vuex';
 import { localizedTexts } from '@/i18n';
 import SessionService from '@/services/session';
 import Login from './Login.vue';
 import router from '../../router';
+import { StateManager } from '../../state';
```

Comment:
At some point you should perhaps put in some aliases for the paths like the @ you use above for the src folder.  I see you are using @ quite often below, so that is good.  Sorry for the nitpick.